templates/component-definitions/devtools_cloud_gov/component-definition.json

```
@@ -0,0 +1,207 @@
+{
+  "component-definition": {
+    "uuid": "d8c6b192-f2c1-4434-8307-d8aed1826fe6",
+    "metadata": {
+      "title": "Cloud.gov Best Practices for DevTools-based applications.",
+      "last-modified": "2024-08-15T01:28:29.538446+00:00",
+      "version": "0.0.1",
+      "oscal-version": "1.1.2"
+    },
+    "components": [
+      {
+        "uuid": "f1815d23-9f15-4824-b7c3-b5e4df26f7cc",
+        "type": "policy",
+        "title": "DevTools Cloud.gov",
+        "description": "A set of testable best practices for cloud.gov hosted apps.",
+        "props": [
+          {
+            "name": "Rule_Id",
+            "value": "ssh-access-disabled",
+            "remarks": "rule_ssh_access_disabled_1"
+          },
+          {
+            "name": "Rule_Description",
+            "value": "Production spaces should disable ssh access",
+            "remarks": "rule_ssh_access_disabled_1"
+          },
+          {
+            "name": "Parameter_Id",
+            "value": "gov.cloud.space-names",
+            "remarks": "rule_ssh_access_disabled_1"
+          },
+          {
+            "name": "Parameter_Description",
+            "value": "The cloud.gov spaces in which ssh access should be disabled",
+            "remarks": "rule_ssh_access_disabled_1"
+          },
+          {
+            "name": "Parameter_Value_Alternatives",
+            "value": "TODO - update to comma separated list of production spaces",
+            "remarks": "rule_ssh_access_disabled_1"
+          },
+          {
+            "name": "Rule_Id",
+            "value": "ssh-access-disabled",
+            "remarks": "rule_ssh_access_disabled_2"
+          },
+          {
+            "name": "Rule_Description",
+            "value": "Production spaces should disable ssh access",
```

suggestion (non-blocking): I'm not sure I'm reading this correctly (or that it matters at this stage) but one of these `rule_ssh_access_disabled_2`'s is not like the other.
```suggestion
            "value": "Production orgs should disable ssh access",
```

---

Will update this the next time the file is touched.